### PR TITLE
Light 563

### LINKED ISSUE #563: Activity indicator for ERC 2022
The rover for ERC 2022 needs an activity blinkenlight that:

- Flashes for >5 seconds before the rover performs an action
- Blinks/flashes to attract attention

### PR DESCRIPTION
**Pull request by vexandmore**
# Assignee Section

## Description
- Adds code so the wheel teensy can control the activity light.
- Improve the organization of the debug mode for serial

### Steps for Testing
1. Upload the wheel teensy code, ensure the status indicator light is plugged in
2. Switch to this branch on the on-board computer, and ensure the main repo is in this branch
3. Restart ROS if a branch switch was needed
4. SSH into the rover
5. Issue the following command: ```rostopic pub /rover_command std_msgs/String "data: 'blink_toggle 1'" ```
6. The status indicator light should be flashing red

closes #563

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [ ] Local Test Performed Successfully
- [ ] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

For Pull Requests that do not include code changes, it is not required to perform the tests above.

### CODE REVIEW

**Comment by RealWilliamWells** [comment]
Just tested with both halfs of the indicater light. blink_color and blink_toggle commands work perfectly as expected and the rest of the wheel code works just fine still! Great work!

File: robot/rover/MobilePlatform/src/APA102.cpp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  }
+  for (int i = START_FRAME_SIZE + numOfLEDs * 4; i < bufSize; i++) {
+    buf[i] = 255;
+  }  
+}
+void APA102::send() {
+  //spi->beginTransaction(settingsA);
+  for (int i = 0; i < bufSize; i++) {
+    shiftOut(mosiPin, clockPin, MSBFIRST, buf[i]);
+    //spi->transfer(buf[i]);
```
Comment (RealWilliamWells) [inline robot/rover/MobilePlatform/src/APA102.cpp:30]: These commented out lines should probably be removed

**Review by RealWilliamWells — APPROVED** [review]
Just went over your work. Looks good to me! Just found some commented out lines that should probably be removed, other than that I approve it!

File: robot/rover/MobilePlatform/src/APA102.cpp (hunk 1 above)
Comment (vexandmore) [reply robot/rover/MobilePlatform/src/APA102.cpp]: just committed this change

**Review by VimFreeman — APPROVED** [review]
code looks good and is very clean. good job. 
good to merge since the test passed with will :)

**Review by RealWilliamWells — APPROVED** [review]

**Pull request merged**